staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -1392,6 +1347,13 @@ func TestPatchStatus(t *testing.T) {
 }
 
 func Test_getNodeConditionPredicate(t *testing.T) {
+	// init feature gates, feature spec does not matter since it will be overriden by tests
```

Comment:
Misspelling here of "overridden". And I think you mentioned that the test changes are moved to another pr #90987

Reply:
Good catch, will update after https://github.com/kubernetes/kubernetes/pull/90987 is merge d